### PR TITLE
Update encryption options for calls

### PR DESCRIPTION
**Pull request by streamer45**
#### Summary

Updating the *Encryption-in-transit* section to include details of encryption for calls connections and data.

#### Ticket Link

https://mattermost.atlassian.net/browse/MM-51797

### CODE REVIEW

**Review by jupenur — APPROVED** [review]

File: source/deploy/encryption-options.rst [hunk 1]
```diff
@@ -19,6 +19,12 @@ Mattermost supports TLS encryption including AES-256 with 2048-bit RSA on all da
 
 Connections to Active Directory/LDAP can `optionally be secured with TLS or stunnel </configure/configuration-settings.html#ad-ldap-port>`__.
 
+Connections to calls are secured with a combination of:
+
+  - TLS: the existing WebSocket channel is used to secure the signaling path.
+  - DTLS v1.2 (mandatory): used for initial key exchange. Supports ``TLS_ECDHE_ECDSA_WITH_AES_128_GCM_SHA256`` and ``TLS_ECDHE_ECDSA_WITH_AES_256_CBC_SHA`` algorithms.
```
Comment (cwarnermm) [inline source/deploy/encryption-options.rst:25]:
```suggestion
  - DTLS v1.2 (mandatory): Used for initial key exchange. Supports ``TLS_ECDHE_ECDSA_WITH_AES_128_GCM_SHA256`` and ``TLS_ECDHE_ECDSA_WITH_AES_256_CBC_SHA`` algorithms.
```

File: source/deploy/encryption-options.rst [hunk 2]
```diff
@@ -19,6 +19,12 @@ Mattermost supports TLS encryption including AES-256 with 2048-bit RSA on all da
 
 Connections to Active Directory/LDAP can `optionally be secured with TLS or stunnel </configure/configuration-settings.html#ad-ldap-port>`__.
 
+Connections to calls are secured with a combination of:
+
+  - TLS: the existing WebSocket channel is used to secure the signaling path.
```
Comment (cwarnermm) [inline source/deploy/encryption-options.rst:24]:
```suggestion
  - TLS: The existing WebSocket channel is used to secure the signaling path.
```

File: source/deploy/encryption-options.rst [hunk 3]
```diff
@@ -19,6 +19,12 @@ Mattermost supports TLS encryption including AES-256 with 2048-bit RSA on all da
...
+  - DTLS v1.2 (mandatory): used for initial key exchange. Supports ``TLS_ECDHE_ECDSA_WITH_AES_128_GCM_SHA256`` and ``TLS_ECDHE_ECDSA_WITH_AES_256_CBC_SHA`` algorithms.
+  - SRTP (mandatory): used to encrypt all media packets (i.e. those containing voice or screen share). Supports ``AEAD_AES_128_GCM`` and ``AES128_CM_HMAC_SHA1_80`` algorithms.
```
Comment (cwarnermm) [inline source/deploy/encryption-options.rst:26]:
```suggestion
  - SRTP (mandatory): Used to encrypt all media packets (i.e. those containing voice or screen share). Supports ``AEAD_AES_128_GCM`` and ``AES128_CM_HMAC_SHA1_80`` algorithms.
```

**Review by cwarnermm — APPROVED** [review]
Thanks, @streamer45!

**Pull request merged**